src/cohortManager.js

```diff
@@ -0,0 +1,84 @@
+class CohortManager {
+  constructor() {
+    this.cohortData = []
+  }
+
+  create(cohort) {
+    const obj = {
+      cohortName: cohort,
+      students: []
+    }
+
+    this.cohortData.push(obj)
+    return this.cohortData
+  }
+
+  searchCohort(cohort) {
+    const searchResult = this.cohortData.filter((obj) => {
+      return obj.cohortName === cohort
+    })
+
+    if (searchResult[0] === undefined) {
+      throw new Error("The cohort doesn't exist")
+    }
+    return searchResult[0]
+  }
+
+  addStudent(firstName, lastName, githubUsername, email, cohort) {
```

Comment:
did you need all the student info? would the ID alone not have been enough to identify each student (provided all ID's were unique numbers)

Reply:
I think the student info was a requirement to add, including the studentID